chaoscenter/web/src/views/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { FontVariation } from '@harnessio/design-system';
+import { Layout, Container, FormInput, ButtonVariation, Text, Button } from '@harnessio/uicore';
+import type { UseMutateFunction } from '@tanstack/react-query';
+import { Formik, Form } from 'formik';
+import { Icon } from '@harnessio/icons';
+import * as Yup from 'yup';
+import type { ResetPasswordOkResponse, ResetPasswordMutationProps } from '@api/auth/index.ts';
+import { useStrings } from '@strings';
+
+interface ResetPasswordViewProps {
+  handleClose: () => void;
+  resetPasswordMutation: UseMutateFunction<
+    ResetPasswordOkResponse,
+    unknown,
+    ResetPasswordMutationProps<never>,
+    unknown
+  >;
+  username: string | undefined;
+}
+interface ResetPasswordFormProps {
+  password: string;
+  reEnterPassword: string;
+}
+
+export default function ResetPasswordView(props: ResetPasswordViewProps): React.ReactElement {
+  const { handleClose, resetPasswordMutation, username } = props;
+  const { getString } = useStrings();
+
+  function isSubmitButtonDisabled(values: ResetPasswordFormProps): boolean {
+    if (values.password === '' || values.reEnterPassword === '') {
+      return true;
+    }
+    if (values.password !== values.reEnterPassword) {
+      return true;
+    }
+    return false;
+  }
+
+  function doesNewPasswordMatch(values: ResetPasswordFormProps): boolean {
+    if (values.password !== values.reEnterPassword) {
+      return false;
+    }
+    return true;
+  }
+
+  function handleSubmit(values: ResetPasswordFormProps): void {
+    doesNewPasswordMatch(values) &&
+      username &&
+      resetPasswordMutation(
+        {
+          body: {
+            username: username,
+            oldPassword: '',
```

Comment:
why you need old password in resetting password

Reply:
reset and update passwords uses common interface in BE, that's why passing empty string. It's handled in BE.
cc @Saranya-jena